core/spec/models/spree/order_spec.rb

```diff
@@ -538,6 +538,16 @@ def merge!(other_order, user = nil)
       expect(order.available_payment_methods).to include(payment_method)
     end
 
+    it "does not include inactive payment methods" do
```

Comment:
This spec surely wants to have a `pending` call to make sure this commit doesn't break the suite, no?